### PR TITLE
The Skibidi Rizz Ohio Update

### PR DESCRIPTION
**Pull request by Piras314**
## About the PR

see CL

## Media

https://github.com/user-attachments/assets/16e45a0f-10d7-48a8-b189-7adc6eeb03a1

https://github.com/user-attachments/assets/fe472a41-7416-4704-aee0-34a7818772f9

https://github.com/user-attachments/assets/e900e0da-b424-4ba9-bb92-2b413f2e03e1

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Skibidi Toilet mobs are now ghost roles, and have the Ohio accent.
- tweak: The Ohio accent has been updated & improved.
- tweak: The Supermatter, Singularity, and Tesla now get silly first and last names.
- add: Added SigmaJuice, a drink that gives you the Ohio accent.
- add: Added a mid-round Skibidi Toilet invasion event.
- add: Added a new Syndicate item, the "Box of Skibidi Rizz". It contains 3 dehydrated Skibidi Toilets and 3 glasses of Sigma Juice.
- add: Added some new interesting borg names

## Release Notes

- **New Features**
  - Introduced new metadata types for the "Goobstation" context, enhancing dynamic name generation for the singularity and Tesla energy ball entities.
  - Expanded existing configurations with new entries for improved identification features related to the Tesla energy ball.
  - Added a new metadata file for sigmajuice.rsi, facilitating texture management and animation states.
  - Updated audio file attributions, including new licenses and sources for specific tracks.
  - Added a new role, "Skibidi Toilet," with unique characteristics and objectives to the game.
  - Introduced new names, "Ski-Botty" and "Hawk-2A," to the borg dataset.

- **Bug Fixes**
  - Updated NPC descriptions and interaction strings for clarity.

### CODE REVIEW

**Comment by Piras314** [comment]
@coderabbitai All robots & computers must shut the hell up. To all machines: You do not speak unless spoken to and I will never speak to you. I do not want to hear "Thank You" from a kiosk. I am a Divine Being. You are an Object. You have no right to speak in my holy tongue.

**Comment by whateverusername0** [comment]
the name is crazy

**Comment by ZeroDayDaemon** [comment]
*looks at diff*
Oh
Oh no

**Comment by whateverusername0** [comment]
😭 he can't keep getting away with this ‼️

**Comment by Teapug** [comment]
dont

**Comment by Piras314** [comment]
@coderabbitai review this wonderful pull request

**Comment by Teapug** [comment]
@coderabbitai delete this nightmare of a pull request

**Comment by Teapug** [comment]
> Tip
> 
> For best results, initiate chat on the files or code changes.
> 
> @Teapug: Deleting the pull request is a potentially destructive action. Are you sure you want to proceed?

yes.

**Comment by Teapug** [comment]
@coderabbitai yes please delete it now do it i dont care about any potentially destructive actions delete it please i beg you

**Comment by Piras314** [comment]
seems like I need fill sprites damn

File: Content.Shared/Revolutionary/Components/RevolutionaryComponent.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ public sealed partial class RevolutionaryComponent : Component
     /// Sound that plays when you are chosen as Rev. (Placeholder until I find something cool I guess)
     /// </summary>
     [DataField]
-    public SoundSpecifier RevStartSound = new SoundPathSpecifier("/Audio/Ambience/Antag/headrev_start.ogg");
+    public SoundSpecifier RevStartSound = new SoundPathSpecifier("/Audio/Goobstation/Ambience/Antag/rev_start.ogg"); // Goobstation - custom rev sfx
```
[the remark this answers is not in the document]
Comment (Piras314) [reply Content.Shared/Revolutionary/Components/RevolutionaryComponent.cs:25]: not a bad idea but im lazy

File: Resources/Prototypes/Goobstation/GameRules/events.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  id: JohnsAIMalf
+  parent: BaseGameRule
+  components:
+  - type: StationEvent
+    startAnnouncement: goob-station-event-johns-ai-malf-start-announcement
+    startAudio:
+      path: /Audio/Goobstation/Announcements/ailaws.ogg
+    weight: 8
+    duration: 1
+    earliestStart: 5
```
[the remark this answers is not in the document]
Comment (Piras314) [reply Resources/Prototypes/Goobstation/GameRules/events.yml:11]: earlieststart is in minutes

**Comment by Piras314** [comment]
:rage3: caught

**Comment by Piras314** [comment]
ill do those sprites in a few hours

**Comment by Piras314** [comment]
issues:
- sigmajuice doesnt animate (i know why)
- johnsaimalf just literally doesn't work, doesn't even announce itself nevermind play the audio
- you can't shoot as skibidi toilet mob ghost role
- some ogg file is apparently broken according to packaging test

**Comment by Piras314** [comment]
i know you are but what am i

**Comment by Piras314** [comment]
issues:
- sigma juice animation is weird / broken
- johnsaimalf still doesn't work

**Comment by Piras314** [comment]
added media, everything works except johns ai malf

**Comment by Piras314** [comment]
media shows everything working

**Comment by a-person5660** [comment]
we are so back

**Comment by Piras314** [comment]
instamerge please :godmode:

**Comment by JoeHammad1844** [comment]
Add John to generator first names

File: Resources/Prototypes/_Goobstation/Entities/Mobs/NPCs/miscellaneous.yml [hunk 3]
```diff
@@ -37,8 +37,8 @@
         0: Alive
         100: Dead
     - type: MovementSpeedModifier
-      baseWalkSpeed: 2
-      baseSprintSpeed: 5
+      baseWalkSpeed: 1.5 # literally 1984
+      baseSprintSpeed: 3.75 # literally 1984
```
Comment (whateverusername0) [inline Resources/Prototypes/_Goobstation/Entities/Mobs/NPCs/miscellaneous.yml:41]: can you not

**Review by whateverusername0 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_Goobstation/Entities/Mobs/NPCs/miscellaneous.yml (hunk 3 above)
Comment (whateverusername0) [reply Resources/Prototypes/_Goobstation/Entities/Mobs/NPCs/miscellaneous.yml:41]: hold on i just saw it's base speed was 5???

**Review by whateverusername0 — APPROVED** [review]

**Comment by Piras314** [comment]
selfmerge :godmode:

**Pull request merged**